alchemist/alchemist-incarnation-protelis/build.gradle

```diff
@@ -9,10 +9,20 @@ dependencies {
         project(':alchemist-engine'),
         project(':alchemist-loading'),
     )
+    compile 'org.eclipse.xtend:org.eclipse.xtend.lib:2.10.0'
 }
 
 configurations {
     compile {
         exclude group: 'org.protelis', module: 'protelis-test'
     } 
-}
\ No newline at end of file
+}
+buildscript {
+    repositories {
+        jcenter()
```

Comment:
Absolutely forbidden to use anything but Maven Central.

Reply:
Same here, automagically added by the plugin when configuring project structure, I'll fix it as soon as possible.